packages/core/src/domain/configuration/configuration.ts

```diff
@@ -59,6 +59,8 @@ export interface InitConfiguration {
   internalAnalyticsSubdomain?: string
 
   telemetryConfigurationSampleRate?: number
+
+  usePciIntake?: boolean
```

Comment:
**💬 suggestion:** ‏Since it is only log related, we should move this to the `LogsInitConfiguration` interface. You could change the endpoint builder code to accept `InitConfiguration & { usePciIntake?: boolean }` as parameter. WDYT?

Reply:
It makes sense since this param is only for logs config.
Done ✔️